install/installer/pkg/config/v1/config.go

```diff
@@ -337,28 +331,11 @@ type WorkspaceTemplates struct {
 	Regular    *corev1.Pod `json:"regular"`
 }
 
-type PersistentVolumeClaim struct {
-	// Size is a size of persistent volume claim to use
-	Size resource.Quantity `json:"size" validate:"required"`
-
-	// StorageClass is a storage class of persistent volume claim to use
-	StorageClass string `json:"storageClass"`
-
-	// SnapshotClass is a snapshot class name that is used to create volume snapshot
-	SnapshotClass string `json:"snapshotClass"`
-}
-
 type Workspace struct {
 	Runtime   WorkspaceRuntime    `json:"runtime" validate:"required"`
 	Resources Resources           `json:"resources" validate:"required"`
 	Templates *WorkspaceTemplates `json:"templates,omitempty"`
 
-	// PrebuildPVC is the struct that describes how to setup persistent volume claim for prebuild workspace
-	PrebuildPVC PersistentVolumeClaim `json:"prebuildPVC" validate:"required"`
-
-	// PVC is the struct that describes how to setup persistent volume claim for regular workspace
-	PVC PersistentVolumeClaim `json:"pvc" validate:"required"`
-
```

Comment:
note: need to clean up the config here as well: https://github.com/gitpod-io/ops/blob/main/deploy/workspace/installer-values.yaml